### PR TITLE
Docs: added more content and fixed spelling errors

### PR DESCRIPTION
**Pull request by ostepizza**
As the title says, despite the code-freeze, I figured we needed to make some slight changes and improvements to the docs before the handover. Comment below if there are more additions/changes necessary/wanted.

### CODE REVIEW

**Review by sebmid — APPROVED** [review]
FINAL PULL REQUEST RIP BOZO

https://github.com/TheAtlasRepository/ImgPDF2Map/assets/89196146/fc433ca8-a4d4-4bbd-b86c-98a335048010

File: docs/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# DOCS :D
+# ImageTopMap Documentation
```
Comment (Trivinyx) [inline docs/README.md]: top?

**Review by Trivinyx — APPROVED** [review]
All else good, just wondering about the commented line.

File: docs/README.md (hunk 1 above)
Comment (ostepizza) [reply docs/README.md]: gahhdah

**Review by sebmid — APPROVED** [review]
FINAL FINAL FINAL REVIEW

**Comment by ostepizza** [comment]
> FINAL FINAL FINAL REVIEW

WITH THIS I SQUASH AND MERGE THE FINAL PR OF BACHELORS??? BELOW IS ME DAY AFTER BACHELORS IS DELIVERED (COINCIDENTALLY MAY 17TH??)

https://github.com/TheAtlasRepository/ImgPDF2Map/assets/32636970/2db3f2ff-3eca-4ad9-92d0-866c22fe5f9d

**Pull request merged**